package.json

```diff
@@ -13,58 +13,58 @@
   "author": "Evan Willhite <evan@fourkitchens.com>",
   "license": "MIT",
   "dependencies": {
-    "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.6.4",
-    "@babel/polyfill": "^7.7.0",
-    "@babel/preset-env": "^7.6.3",
-    "@storybook/addon-a11y": "5.2.6",
-    "@storybook/addon-actions": "5.2.6",
-    "@storybook/addon-links": "5.2.6",
-    "@storybook/addons": "5.2.6",
-    "@storybook/react": "5.2.6",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.4",
+    "@babel/polyfill": "^7.8.3",
+    "@babel/preset-env": "^7.8.4",
+    "@storybook/addon-a11y": "^5.3.13",
+    "@storybook/addon-actions": "^5.3.13",
+    "@storybook/addon-links": "^5.3.13",
+    "@storybook/addons": "^5.3.13",
+    "@storybook/react": "^5.3.13",
     "@storybook/storybook-deployer": "^2.8.1",
     "add-attributes-twig-extension": "^0.1.0",
-    "autoprefixer": "^9.6.5",
+    "autoprefixer": "^9.7.4",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.6",
     "babel-preset-minify": "^0.5.0",
     "bem-twig-extension": "^0.1.1",
     "breakpoint-sass": "^2.7.1",
     "clean-webpack-plugin": "^3.0.0",
-    "concurrently": "^5.0.0",
-    "cross-env": "6.0.3",
-    "css-loader": "^3.2.0",
-    "eslint": "^6.5.1",
+    "concurrently": "^5.1.0",
+    "cross-env": "7.0.0",
```

Comment:
Looks like this one's the last one that is pinned to a specific version. And again, running `yarn add cross-env` puts the caret in front of it.